main.tf

```diff
@@ -0,0 +1,192 @@
+module "label" {
+  source     = "github.com/cloudposse/terraform-terraform-label.git?ref=master"
+  attributes = "${var.attributes}"
+  delimiter  = "${var.delimiter}"
+  name       = "${var.name}"
+  namespace  = "${var.namespace}"
+  stage      = "${var.stage}"
+  tags       = "${var.tags}"
+}
+
+# ECR repository
+resource "aws_ecr_repository" "app" {
```

Comment:
Move out of the module. We should be using our `terraform-aws-ecr` module for this. Invoke that module in `examples/`